src/gui/editorwidgets/qgshtmlwidgetwrapper.cpp

```diff
@@ -52,12 +52,14 @@ void QgsHtmlWidgetWrapper::initWidget( QWidget *editor )
   mWidget->setZoomFactor( horizontalDpi / 96.0 );
 
   QWebPage *page = mWidget->page();
+  page->settings()->setAttribute( QWebSettings::DeveloperExtrasEnabled, true );
```

Comment:
Is this required by `javaScriptWindowObjectCleared` below?

Reply:
No, the fix works without it. I moved it because I thought it was better to do it when initializing widget, instead of doing it when setting the feature.

According to the documentation, I think we could remove it 

> Enables extra tools for Web developers. Currently this enables the "Inspect" element in the context menu as well as the use of QWebInspector which controls the web inspector for web site debugging. This is disabled by default.

I can remove it, I'm just afraid about some potential side effect I'm not aware of.

Reply:
I removed it because I do think its useless :crossed_fingers: